.php_cs.dist

```diff
@@ -82,10 +82,15 @@ $config = PhpCsFixer\Config::create()
 // special handling of fabbot.io service if it's using too old PHP CS Fixer version
 if (false !== getenv('FABBOT_IO')) {
     try {
-        PhpCsFixer\FixerFactory::create()
-            ->registerBuiltInFixers()
-            ->registerCustomFixers($config->getCustomFixers())
-            ->useRuleSet(new PhpCsFixer\RuleSet($config->getRules()));
+        $config = PhpCsFixer\FixerFactory::create()
+            ->registerBuiltInFixers();
+        $config->useRuleSet(new PhpCsFixer\RuleSet(array_filter(
```

Comment:
This will detect unknown rules but not unknown/invalid config for known rules. Shall we go further to cover all cases?

Reply:
Fixing having unknown rules solves current problem, as we agree how to solve it let's go with invalid config then - or is there any clean solution how to do it in `array_filter`?

Reply:
you could create a fixer from rule (`braces` -> `BracesFixer`) and then call `->configure(...)` to know if it's failing with config or not